sunpycube/cube/NDCube.py

```diff
@@ -401,3 +414,130 @@ def _plot_1D_cube(cube, unit=None, origin=0):
     plot = plt.plot(cube.pixel_to_world(
         [u.Quantity(np.arange(cube.data.shape[0]), unit=u.pix)], origin=origin)[0].to(unit), cube.data)
     return plot
+
+
+class NDCubeSequence(object):
+    """
+    Class representing list of cubes.
+
+    Attributes
+    ----------
+    data_list : `list`
+        List of cubes.
+
+    meta : `dict` or None
+        The header of the NDCubeSequence.
+
+    common_axis: `int` or None
+        The data axis which is common between the NDCubeSequence and the Cubes within.
+        For example, if the Cubes are sequenced in chronological order and time is
+        one of the zeroth axis of each Cube, then common_axis should be se to 0.
+        This enables the option for the NDCubeSequence to be indexed as though it is
+        one single Cube.
+    """
+
+    def __init__(self, data_list, meta=None, common_axis=0, **kwargs):
+        self.data = data_list
+        self.meta = meta
+        self.common_axis = common_axis
+        self.time = kwargs.get('time', None)
+
+    def __getitem__(self, item):
+        if item is None or (isinstance(item, tuple) and None in item):
+            raise IndexError("None indices not supported")
+        return cu.get_cube_from_sequence(self, item)
+
+    def plot(self, *args, **kwargs):
+        i = ani.ImageAnimatorNDCubeSequence(self, *args, **kwargs)
+        return i
+
+    def explode_along_axis(self, axis=None):
+        """
+        Creates one less Dimensional data.
```

Comment:
Suggested rewrite: Separates slices of NDCubes in sequence along a given cube axis into (N-1)DCubes.